decadog/src/command/sprint.rs

```diff
@@ -220,8 +220,12 @@ fn finish_sprint(settings: &Settings) -> Result<(), Error> {
     let repository = client.get_repository()?;
     let sprint = client.get_sprint(&repository, &open_milestone)?;
 
+    let mut points_done_in_sprint: u32 = 0;
+    let mut points_done_out_of_sprint: u32 = 0;
+
     println!();
     println!("{}", "Issues closed in the sprint timeframe:".bold());
+    let mut issue_is_planned: bool;
```

Comment:
As Rust's pattern matching is so great, it's a nice idea to use `enum`s rather than bools. The best benefit is readability - there's a good article here: https://www.reddit.com/r/rust/comments/bm6s3t/rust_patterns_enums_instead_of_booleans/

I would write this as:

```rust
enum Planning {
    Planned,
    Unplanned,
}
...

let mut issue_planning = Planning::Planned;
...
// on some condition
issue_planning = Planning::Unplanned;
...
match issue_planning {
    Planning::Planned => points_done_in_sprint += estimate_value,
    Planning::Unplanned => points_done_out_of_sprint += estimate_value,
}
```

Reply:
Also, this variable should only be scoped on a per-issue basis, you're declaring it outside the loop here